src/clr-angular/forms/select/providers/blur-event-disable.service.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+
+import {Injectable} from "@angular/core";
+import {Observable} from "rxjs/Observable";
+import {Subject} from "rxjs/Subject";
+
+@Injectable()
+export class BlurEventDisableService {
+    private _blurEventDisableChange: Subject<boolean> = new Subject<boolean>();
+
+    get blurEventDisableChange(): Observable<boolean> {
+        return this._blurEventDisableChange.asObservable();
+    }
+
+    enableBlurEvent(): void {
+        this._blurEventDisableChange.next(true);
```

Comment:
`enable()` sets the `DisableService` to `true`.
I'm pretty sure you want either to swap the `true/false`, or rename the service. 😉

Reply:
You can keep the name if you want, but then just use `disabled` in your components too (like [here](https://github.com/vmware/clarity/pull/2223/files/f975e60091eac9ec64b988ea4726ceac58770e2f..ceaccae974b442aa297969ff0ac2d52099f438d4#diff-b8e1571d74b2bc9b2f8106a47b7802cbR25)) so that `true` means "disabled" everywhere.

I just laughed again at [this line](https://github.com/vmware/clarity/pull/2223/files/f975e60091eac9ec64b988ea4726ceac58770e2f..ceaccae974b442aa297969ff0ac2d52099f438d4#diff-f20561bd6e03550368d629f4c6cdd065R29):
```typescript
let blurEnabledService: BlurEventDisableService;
```

Reply:
doesnt matter now. removing the service completely. using the tab key event listener instead like we discussed.